.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   extends: "lob",
+  extends: "lob/migrations",
```

Comment:
Done, but to confirm, this meant copying the .eslintrc file into db/migrations and just having the one line about extending lob/migrations. The other lines can remain within the outermost .eslintrc file and will still be applied with the additional directory specific .eslintrc with the lone lint instructions for migrations. Right?